libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -1438,7 +1438,7 @@ - (void)retrieveUserPhotoIfNeeded:(SFUserAccount *)account {
         SFNetwork *network = [[SFNetwork alloc] initWithEphemeralSession];
         [network sendRequest:request  dataResponseBlock:^(NSData *data, NSURLResponse *response, NSError *error){
             if (error) {
-                [SFSDKCoreLogger w:[self class] format:@"Error while trying to retrieve user photo: %@ %@", (long) error.code, error.localizedDescription];
+                [SFSDKCoreLogger w:[self class] format:@"Error while trying to retrieve user photo: %ld %@", (long) error.code, error.localizedDescription];
```

Comment:
Or the other way it to do:
```objective-c
[SFSDKCoreLogger w:[self class] format:@"Error while trying to retrieve user photo: %@ %@", @(error.code), error.localizedDescription];
```

Reply:
True! I did that to align with similar logging in this file but happy to change it next time I'm visiting that file :-)